GEOSldas_App/util/postproc/ObsFcstAna_stats/postproc_ObsFcstAna.py

```diff
@@ -0,0 +1,537 @@
+
+# Tool to prepocess GEOSldas ObsFcstAna output into monthly sums, sums of squares, and sum of cross-products
+#
+# qliu, amfox, rreichle - May 2025
+
+import numpy as np
+import os
+
+import sys;       sys.path.append('../../shared/python/')
+
+import warnings;  warnings.filterwarnings("ignore")
+
+from datetime                    import datetime, timedelta
+from dateutil.relativedelta      import relativedelta
+from netCDF4                     import Dataset, date2num
+
+from util                        import make_folder
+from helper.compute_monthly_sums import compute_monthly_sums
+from helper.write_nc4            import write_sums_nc4, write_stats_nc4, write_omf_stats_nc4, write_omf_grouped_stats_nc4
+
+class postproc_ObsFcstAna:
+    
+    def __init__(self, exp_list, start_time, end_time, obs_from=0):
+        self.expdir_list   = [item['expdir'] for item in exp_list]
+        self.expid_list    = [item['expid']  for item in exp_list]
+        self.exptag_list   = [item['exptag'] for item in exp_list]
+        self.domain        = exp_list[0]['domain']
+        self.start_time    = start_time 
+        self.end_time      = end_time 
+        self.var_list      = ['obs_obs','obs_obsvar','obs_fcst','obs_fcstvar','obs_ana','obs_anavar']
+        self.tilecoord     = exp_list[0]['tilecoord']
+        self.obsparam_list = [item['obsparam'] for item in exp_list]
+        self.obs_from      = obs_from
+
+    # ---------------------------------------------------------------------------
+        
+    def save_monthly_sum(self, outpath='./'):
+        expdir_list   = self.expdir_list
+        expid_list    = self.expid_list
+        exptag_list   = self.exptag_list
+        domain        = self.domain
+        var_list      = self.var_list
+        tc            = self.tilecoord
+        obsparam_list = self.obsparam_list
+        start_time    = self.start_time
+        end_time      = self.end_time
+
+        start_month =  start_time.replace(day=1,hour=3)
```

Comment:
LANDASSIM_DT and LANDASSIM_T0 should not be hardwired here and in a couple of other places throughout the scripts.  They can be specified by the user as variables in the Main script.   (That is, no need to extract them from the experiment directory.)

Reply:
They are now specified by user. I removed the hard coded part.

Reply:
The Plot_stats_maps.py example script is now missing this specification. i.e missing:
          'da_t0'        : 3,       # first hour of the month 
          'da_dt'        : 10800,   # ObsFcstAna file interval in seconds